combat-trainer.lic

```diff
@@ -2901,6 +2910,10 @@ class GameState
     @summoned_weapons.find { |summoned_skill| summoned_skill['name'] == weapon_skill }
   end
 
+  def weapon_is_summoned?
+    @summoned_weapons.include? weapon_skill
```

Comment:
❓ Does this work? From [looking at the format](https://github.com/rpherbig/dr-scripts/blob/05a2211fbac5b1409bf834d003d2517d104f3726/profiles/Torgro-back.yaml#L11-L12) of `summoned_weapons` I expected syntax like line 2910 (`... ['name']`)

Reply:
Good catch, I'll check this out now, thanks!